drivers/msgbox/msgbox.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2017-2020 The Crust Firmware Authors.
+ * SPDX-License-Identifier: BSD-3-Clause OR GPL-2.0-only
+ */
+
+#include <device.h>
+#include <intrusive.h>
+#include <stdbool.h>
```

Comment:
You should also include `<msgbox.h>` here, so the prototypes for the functions defined here are available.

Reply:
Done